test/alternator/test_table.py

```diff
@@ -110,6 +110,16 @@ def test_create_and_delete_table_very_long_name(dynamodb):
 # Tests creating a table with an invalid schema should return a
 # ValidationException error.
 def test_create_table_invalid_schema(dynamodb):
+    # Creation of a table without AttributeDefinitions is illegal
+    with pytest.raises(ClientError, match='ValidationException.*[aA]ttributeDefinitions'):
+        dynamodb.create_table(
+            TableName='name_doesnt_matter',
```

Comment:
This "name_doesnt_matter" thing is old style ;-) I prefer to use the new_test_table function which picks a unique name for you (and deletes the table if by some miracle, the creation passed). See an example how easy this is in test_create_table_spurious_attribute_definitions